enclave-manager/web/src/client/enclaveManager/KurtosisClientContext.tsx

```diff
@@ -131,3 +138,12 @@ export const useKurtosisClient = (): KurtosisClient => {
 
   return client;
 };
+
+const paramToUrl = (searchParams: URLSearchParams, param: string) => {
+  let paramString = searchParams.get(param);
+  if (paramString === null) return null;
```

Comment:
I'll add eslint config for `curly` -https://eslint.org/docs/latest/rules/curly. Generally I dislike this form of conditional as it's less readable and a source of developer errors. Famous example of why not to do this https://www.synopsys.com/blogs/software-security/understanding-apple-goto-fail-vulnerability-2.html

Reply:
That's fair. Thanks for fixing that!